cmake/TestImpactFramework/LYTestImpactFramework.cmake

```diff
@@ -392,6 +389,23 @@ function(ly_extract_target_dependencies INPUT_DEPENDENCY_LIST OUTPUT_DEPENDENCY_
     set(${OUTPUT_DEPENDENCY_LIST} ${dependencies} PARENT_SCOPE)
 endfunction()
 
+#! ly_get_target_type: retrieves the type of target (either production or test target).
+#
+# \arg:TARGET target to get the type for
+# \arg:TARGET_TYPE the retrieved target type
+function(ly_get_target_type TARGET TARGET_TYPE)
```

Comment:
Not being familiar with the TIAF, this function's name makes me think that it will [fetch the `TARGET`'s `TYPE` property](https://cmake.org/cmake/help/latest/prop_tgt/TYPE.html). I expect it to be:
```cmake
get_target_property(${TARGET_TYPE} ${TARGET} type)
```

Reply:
I see where you're coming from, i will actually namespace the tiaf functions as `o3de_tiaf_XXXX` (instead of `ly_XXXX`) in a separate PR (will leave as-is for now) 👍